src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -85,6 +85,12 @@ Create ``unit-test-configuration.ts`` file based on
 Run ``npm run test`` to execute the unit tests via `Jest
 <https://facebook.github.io/jest/>`_.
 
+If you get errors on all tests, it could be the reason that `Jest
+<https://facebook.github.io/jest/>`_ or something else were updated.
+There are a view ways how you can try to resolve this:
+- Remove all modules and run ``npm install`` again in order to reinstall them
```

Comment:
Maybe you should add the info how to remove the modules, e.g. ``rm -rf dist/ node_modules/ package-lock.json``.

Reply:
I would not recommend removing `package-lock.json`.